examples/models/custom_metrics/customMetrics.ipynb

```diff
@@ -35,26 +35,42 @@
    "cell_type": "code",
```

Comment:
<p>In the PR please commit it as <code>seldonio/model-with-metrics:0.2</code> . </p><p> </p><p>As I can see you are using <code>minikube</code>.  You can build the image as <code>seldonio/model-with-metrics:0.2</code>  using your minikube's docker so it is available right inside it using by executing first</p><pre class="ql-syntax" spellcheck="false">eval $(minikube docker-env)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/2907/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>